e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -464,7 +453,7 @@ az keyvault secret set --vault-name $keyVaultName --name "IOTHUB-EVENTHUB-CONN-S
 az keyvault secret set --vault-name $keyVaultName --name "IOTHUB-EVENTHUB-COMPATIBLE-NAME" --value $eventResourceGroup --output none
 az keyvault secret set --vault-name $keyVaultName --name "IOTHUB-EVENTHUB-CONSUMER-GROUP" --value $consumerGroups --output none
 az keyvault secret set --vault-name $keyVaultName --name "IOTHUB-PROXY-SERVER-ADDRESS" --value $proxyServerAddress --output none
-az keyvault secret set --vault-name $keyVaultName --name "FAR-AWAY-IOTHUB-HOSTNAME" --value "$farHubName.azure-devices.net" --output none
```

Comment:
Was this a bug?

Reply:
Not a bug. Previously the $farHubName was set in the .ps1 to be equal to ResouceGroupName. Now i'm no longer doing it. There is a default in the ARM template and the template will just output the hostname. There is no need ot the concatenation anymore.